spyder_kernels/customize/tests/test_utils.py

```diff
@@ -16,7 +16,7 @@ def test_user_sitepackages_in_pathlist():
     """Test that we include user site-packages in pathlist."""
     if sys.platform.startswith('linux'):
         user_path = 'local'
-    elif sys.platform == 'darwin':
+    elif (sys.platform.startswith('darwin') | sys.platform.startswith('freebsd')):
```

Comment:
I suggest to change this to

```python
elif (sys.platform == darwin or sys.platform.startswith('freebsd')):
```

This conforms to the way we check for Mac in our codebase.

Reply:
Thanks for your fast and helpful response. Of course, I will follow you suggestion about darwin.

If I want to submit a PR against the 2.x branch, should I open a new PR or is it possible to do it in this one?

Reply:
I'm afraid you have to open a new PR against `2.x` and close this one. That's because you didn't use a branch to create your PR but instead pushed directly to `master` (with a branch it'd be a matter of issuing a few git commands to make it derive from `2.x`)